### PR TITLE
Fix issue #156 to return proper message for active job

### PR DESCRIPTION
**Pull request by tiantn**
**What It Does**

**How to Test**

**Review Checklist**
I certify that I have:
- [x] tested my changes
- [ ] added/updated automated tests
- [x] updated the changelog
- [ ] followed the [contribution guidelines](https://github.com/zowe/zowe-cli/blob/master/CONTRIBUTING.md)

**Additional Comments**

### CODE REVIEW

**Review by traeok — APPROVED** [review]
Tested locally and looks good to me, thanks @tiantn!

**Review by std4lqi — APPROVED** [review]
Approve. But trailing space at line 80 in `src/api/JobUtils.ts`?

File: src/api/JobUtils.ts [hunk 1]
```diff
@@ -75,17 +75,20 @@ export class JobUtils {
...
-                jobId: jobDetails.jobid,
-                owner: "*",
-                fileId: spoolFileToDownload.id
-            };
-            const spoolFile = await JobUtils.getSpoolFileContent(connection, option);
-            spoolFileToDownload.contents = spoolFile;
-            fullSpoolFiles.push(spoolFileToDownload);
+        if (jobDetails.spoolFiles) {
+            for (const spoolFileToDownload of jobDetails.spoolFiles) {
+                this.log.debug("Requesting spool files for job %s(%s) spool file ID %d",
```
Comment (t1m0thyj) [inline src/api/JobUtils.ts:80]: Please remove trailing spaces to fix lint warning

**Review by t1m0thyj — APPROVED** [review]
LGTM, thanks @tiantn! Tested locally and it works well 👍 

```
$ zowe zftp ls sfbj TSU06008  
No spool file available.
$ zowe zftp view asbj TSU06008 
Command Error:
No spool files were available for job TSU06008. Try again after waiting a moment if the job is not yet in OUTPUT status.
```

File: src/api/JobUtils.ts [hunk 2]
```diff
@@ -75,17 +75,20 @@ export class JobUtils {
...
-                jobName: jobDetails.jobname,
-                jobId: jobDetails.jobid,
-                owner: "*",
-                fileId: spoolFileToDownload.id
-            };
-            const spoolFile = await JobUtils.getSpoolFileContent(connection, option);
-            spoolFileToDownload.contents = spoolFile;
-            fullSpoolFiles.push(spoolFileToDownload);
+        if (jobDetails.spoolFiles) {
+            for (const spoolFileToDownload of jobDetails.spoolFiles) {
```
Comment (zFernand0) [inline src/api/JobUtils.ts:79]: Wondering if we should push the condition down to the `for-of` loop. 
For example `for (const spoolFileToDownload of jobDetails.spoolFiles ?? []) {`

**Review by zFernand0 — APPROVED** [review]
LGTM! 😋 
I don't think I have anything to add that other reviewers haven't mentioned 😋

**Comment by tiantn** [comment]
Thank you all for your good advice. I just removed the trailing spaces. 😊

**Pull request merged**